models/tsodyks2_synapse.h

```diff
@@ -76,12 +76,12 @@ The following parameters can be set in the status dictionary:
 
 ========  ======  ========================================================
  U        real    Parameter determining the increase in u with each spike
-                  (U1) [0,1], default=0.5
+                  (U1) [0,1], default = 0.5
  u        real    The probability of release (U_se) [0,1],
-                  default=0.5
- x        real    Current scaling factor of the weight, default=U
- tau_fac  ms      Time constant for facilitation, default = 0(off)
- tau_rec  ms      Time constant for depression, default = 800ms
+                  default = U
+ x        real    Current scaling factor of the weight, default = 1.0
+ tau_fac  ms      Time constant for facilitation, default = 0 (off)
+ tau_rec  ms      Time constant for depression, default = 800
```

Comment:
The order of the tau_ lines differs from the quantal synapse. Can you ensure same order everywhere?